Documentation/gettingstarted/minikube-install.rst

```diff
@@ -39,9 +39,10 @@
 .. note::
 
    From minikube v1.12.1+, cilium networking plugin can be enabled directly with
-   ``--network-plugin=cilium`` parameter in ``minikube start`` command. With this
+   ``--cni=cilium`` parameter in ``minikube start`` command. With this
    flag enabled, ``minikube`` will not only mount eBPF file system but also
```

Comment:
Step 4 might not be required either - https://github.com/cilium/cilium/pull/16346#discussion_r641245407?